npm/git-cliff/yarn.lock

```diff
@@ -1422,12 +1422,12 @@ __metadata:
     "@typescript-eslint/parser": "npm:^7.1.0"
     eslint: "npm:^8.57.0"
     execa: "npm:^8.0.1"
-    git-cliff-darwin-arm64: "npm:2.1.0-rc.0"
-    git-cliff-darwin-x64: "npm:2.1.0-rc.0"
-    git-cliff-linux-arm64: "npm:2.1.0-rc.0"
-    git-cliff-linux-x64: "npm:2.1.0-rc.0"
-    git-cliff-windows-arm64: "npm:2.1.0-rc.0"
-    git-cliff-windows-x64: "npm:2.1.0-rc.0"
+    git-cliff-darwin-arm64: "npm:2.2.0"
+    git-cliff-darwin-x64: "npm:2.2.0"
+    git-cliff-linux-arm64: "npm:2.2.0"
+    git-cliff-linux-x64: "npm:2.2.0"
+    git-cliff-windows-arm64: "npm:2.2.0"
+    git-cliff-windows-x64: "npm:2.2.0"
```

Comment:
The versions in `package.json` are bumped automatically via `release.sh` script.

https://github.com/orhun/git-cliff/blob/98647800565f588f656234f5b717d3ce35db3344/release.sh#L17

I'm wondering if we should do the same for `yarn.lock`?

Reply:
That could be possible. This bump happened when I ran `yarn install`. Essentially to achieve the automatic bump that would have to be ran and then committed and pushed.

Reply:
Coming back to this, I'm going to have to retract my above statement. You will have to run `yarn install` in the `npm/git-cliff` folder yourself when making a commit such as [chore(release): prepare for v2.2.0
](https://github.com/orhun/git-cliff/commit/bd98caf95351d89a9f4f66e24b6d9217952621ab). That is the proper moment to update the lock file.

Reply:
Added in a21d221